sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -502,52 +505,6 @@ export class CertificateClient {
     }
   }
 
-  /**
```

Comment:
This is a lint I'd like us not to use.  The functions are grouped together because the two helper functions help the public function in the group of three.

By sorting private and public differently, we lose the natural organization of the functionality.

Reply:
I agree with you. I also think the reason why you see this valuable is enough of an argument to disable this rule at the main configuration. Thoughts? @willmtemple @xirzec

Reply:
I reverted this order.